app/assets/stylesheets/base/_base.scss

```diff
@@ -0,0 +1,15 @@
+// Bitters 1.1.0
+// http://bitters.bourbon.io
+// Copyright 2013-2015 thoughtbot, inc.
+// MIT License
+
+@import "variables";
+
+// Neat Settings -- uncomment if using Neat -- must be imported before Neat
```

Comment:
Since we've uncommented the line below, how would you feel about removing this comment?